docs/sqlalchemy.rst

```diff
@@ -207,6 +207,7 @@ system <sa:orm_declarative_mapping>`:
     ...     quote_ft = sa.Column(sa.String)
     ...     even_more_details = sa.Column(sa.String, crate_columnstore=False)
     ...     created_at = sa.Column(sa.DateTime, server_default=sa.func.now())
+    ...     uuid = sa.Column(sa.String, crate_autogenerate_uuid=True)
```

Comment:
Doesn't `id = sa.Column(sa.String, primary_key=True, server_default=func.gen_random_text_uuid())` already work? Why do we need a custom property?

Reply:
You are having a valid point here. Maybe the `crate_autogenerate_uuid` should be a table- or dialect-level parameter instead, to cover situations like outlined in GH-575, which is a slightly different topic, and can be deferred to another patch.

If the user can write their own model definitions, your proposal to use `server_default=func.gen_random_text_uuid()` indeed would be completely satisfactory.

So, the outcome for this topic could be essentially to cover that detail about `func.gen_random_text_uuid()` within the documentation and software tests at
- docs/sqlalchemy.rst
- src/crate/client/sqlalchemy/tests/create_table_test.py

and not add any specific functionality for this on the column level. Apologies that GH-533 was giving wrong guidance here.